clang/lib/Frontend/CompileJobCacheKey.cpp

```diff
@@ -126,6 +157,8 @@ canonicalizeForCaching(llvm::cas::ObjectStore &CAS, DiagnosticsEngine &Diags,
       FrontendOpts.DisableCachedCompileJobReplay;
   FrontendOpts.DisableCachedCompileJobReplay = false;
   FrontendOpts.IncludeTimestamps = false;
+  Opts.PathPrefixMappings = std::move(FrontendOpts.PathPrefixMappings);
+  FrontendOpts.PathPrefixMappings.clear();
```

Comment:
I think technically this is okay for `std::vector`, but I wonder if we should change it to

```c++
std::swap(Opts.PathPrefixMappings, FrontendOpts.PathPrefixMappings);
```

So it can't accidentally cause issues if we ever change the type to something that doesn't guarantee what its state is after a move?

Reply:
Done in https://github.com/apple/llvm-project/pull/5965